test/ostruct/test_ostruct.rb

```diff
@@ -63,6 +63,8 @@ def test_inspect
   end
 
   def test_frozen
+    return if RUBY_ENGINE == 'truffleruby' # https://github.com/oracle/truffleruby/issues/2093
```

Comment:
Since this has been fixed by @bjfish inhttps://github.com/oracle/truffleruby/issues/2093 and the CI uses `truffleruby-head`, it should work fine without this change, @marcandre can you try?

Reply:
👍